default.nix

```diff
@@ -310,15 +310,16 @@ in rec {
         mv $NIX_BUILD_TOP/temp "$PWD/deps/${pname}"
         cd $PWD
 
-        ln -s ${deps}/deps/${pname}/node_modules "deps/${pname}/node_modules"
-
+        if [ -d ${deps}/deps/${pname}/node_modules ]; then
+          cp -r ${deps}/deps/${pname}/node_modules "deps/${pname}/node_modules"
+          chmod -R +w "deps/${pname}/node_modules"
+        fi
```

Comment:
Since we're already copying in all of `node_modules`, we don't really need the modules package for runtime anymore. This link still creates a dependency though, so I'm just copying this too.